src/Sarif/Writers/SarifLogger.cs

```diff
@@ -447,12 +447,16 @@ private void CaptureArtifact(ArtifactLocation fileLocation)
                 catch (ArgumentException) { } // Unrecognized encoding name
             }
 
+            HashData hashData = null;
+            AnalysisTargetToHashDataMap?.TryGetValue(fileLocation.Uri.LocalPath, out hashData);
+
             // Ensure Artifact is in Run.Artifacts and ArtifactLocation.Index is set to point to it
             int index = _run.GetFileIndex(
                 fileLocation,
                 addToFilesTableIfNotPresent: _persistArtifacts,
                 _dataToInsert,
-                encoding);
+                encoding,
+                hashData: hashData);
```

Comment:
>hashData: hashData [](http://example.com/codeflow?start=16&length=18)

it's not valuable to repeat a local name that matches the name of the parameter, no additional information is provided, we've just consumed more space.

Reply:
fixed here and in all the other places in this file.